spec/unit/installer/analyzer/sandbox_analyzer_spec.rb

```diff
@@ -83,6 +84,11 @@ module Pod
         @sandbox.stubs(:predownloaded?).returns(true)
         @analyzer.send(:pod_changed?, 'BananaLib').should == true
       end
+
+      it 'considers a changed Pod whose dependency is changed' do
+        @analyzer.stubs(:sandbox_dependency).returns(['BananaLib (from `https://github.com/grafiszti/bananalib.git`)'])
+        @analyzer.send(:pod_changed?, 'BananaLib').should == true
```

Comment:
have you verified this test fails without your change?

Reply:
yes, i have verified it, although the test failed without my change. but i think it's not right, so i have modified the test, please review it again, thanks~